crates/biome_grit_parser/src/constants.rs

```diff
@@ -50,6 +50,8 @@ pub(crate) const NOT_SET: TokenSet<GritSyntaxKind> = token_set![NOT_KW, T![!]];
 
 pub(crate) const REGEX_SET: TokenSet<GritSyntaxKind> = token_set![GRIT_REGEX, GRIT_SNIPPET_REGEX];
 
+pub(crate) const SUPPORTED_ENGINE_SET: TokenSet<GritSyntaxKind> = token_set![T![biome]];
```

Comment:
```suggestion
pub(crate) const SUPPORTED_ENGINE_SET: TokenSet<GritSyntaxKind> = token_set![T![biome], T![marzano]];
```

`marzano` token is missing